metacat-main/src/main/java/com/netflix/metacat/main/manager/CatalogManager.java

```diff
@@ -114,17 +124,23 @@ private void loadCatalog(final File file) throws Exception {
         log.info("-- Loading catalog {} --", file);
         final Map<String, String> properties = new HashMap<>(this.loadProperties(file));
 
-        final String connectorType = properties.remove("connector.name");
+        final String connectorType = properties.remove(MetacatCatalogConfig.Keys.CONNECTOR_NAME);
         Preconditions.checkState(
             connectorType != null,
             "Catalog configuration %s does not contain connector.name",
             file.getAbsoluteFile()
         );
 
-        final String catalogName = Files.getNameWithoutExtension(file.getName());
-        final ConnectorContext connectorContext = new ConnectorContext(catalogName, config, registry, properties);
-        this.connectorManager.createConnection(catalogName, connectorType, connectorContext);
-        log.info("-- Added catalog {} using connector {} --", catalogName, connectorType);
+        // Catalog shard name should be unique. Usually the catalog name is same as the catalog shard name.
```

Comment:
This seems overly complex why don't we just have 3 properties:
`catalog.name`,
`catalog.shardName` (optional)
`catalog.default` (default false)